### DISCUSSION TITLE
Feedback - v20220514 - conclusion

### DISCUSSION THREAD

**Discussion by edzob** [JRBliekendaal/master-thesis#213] [General]
1. ” was plead to ’Schippers’ " --> add role . for example "was plead to the Dutch Minister of ... 'Schippers' 
3. "A responsive and adaptive government is needed to deal with this. We need to create public organisations that can cope with or even seize opportunities in a dynamic difficult, unpredictable environment." -> Merge to for example : "We need to create public organisations, that are responsive and adaptive, to cope with or even seize opportunities in a dynamic difficult, unpredictable environment.
5. "This continuous change confronts" -> "This speed of change confronts"
7. "using Architecture in different" -> "using Enterprise Architecture in different"
9. "On multiple occasions, the improvement program mentions using Architecture in different appearances (Enterprise Architecture, Nederlandse Overheids Referentie Architectuur (NORA) and Enterprise Architecture Rijksdienst (EAR)) supporting the improvements"  ->  "The improvement program positions Enterprise Architecture supportive to the proposed improvements, specifically the Nederlandse Overheids Referentie Architectuur (NORA) and the Enterprise Architecture Rijksdienst (EAR).
15. "The Dutch public sector wants to change to be more adaptive and responsive. " -> "The Dutch public sector wants to change toward to be more adaptive and responsive. "
18. "It was proposed to use antifragile from Taleb (2012) to deal with disruptive change." -> proposed by who? It is a new paragraph. "The improvement programm proposed to use antifragile from Taleb (2012) to deal with disruptive change."
23. Misschien de zin over taleb een paar zinnen omhoog zodat de flow van EA clean is en in 1 paragraph kan,
24. Tabel: wat bepaald de volgorde van de termen? Ze zijn niet eerst gesorteerd op category en daarna op alphabet. Maar dat zijn we de enige 2 dimensies die de lezer ziet.
26. “The topic of antifragile is still relatively young, and as far as we have been able to find, it has not been used in practice yet in the context of the (Dutch) public sector.” ->  “The concept of antifragile is relatively young, and as far as we have been able to find, it has not been used in practice in the context of the public sector.“  … yet geeft extra betekenis die niet nodig is.. Denk ik. .en het is of Dutch public sector of public sector. Gezien het een subdomain is en je een stelling neemt.
33. “How could we ensure that we have done everything to remove possible subjectivity?” -> rephrase since this is never possible.
36. “With using the triangulation method, we have minimised possible subjectivity by using multiple research tools “  -> By using / By applying  .. reduced subjectivity .. NB. wees voorzichting met deze stellingnames. :)
40. “We performed a literature study” -> rephrase .. perhaps the research process applied is …. 
42. “The possible success factors were also selected (the last seven), and we dropped the ones with only occurrence.” -> This is your final sentence of your exec summ..
43. People will read the begin and end. And then decide to read the middle. Is this wise?

**Comment by JRBliekendaal** [comment]
@edzob 

"It was proposed to use antifragile from Taleb (2012) to deal with disruptive change." -> proposed by who? It is a new paragraph. "The improvement programm proposed to use antifragile from Taleb (2012) to deal with disruptive change."

It was not the improvement program... It was another researcher in the public sector. So I combined the improvement program (the I-Column of the public sector) together with the wishes of the public sector (politics) and a dutch public sector researcher.
